### DISCUSSION TITLE
Disable Email Notifications for Developer Portal

### DISCUSSION THREAD

**Discussion by Mary-Kwiatkowski** [Azure/api-management-developer-portal#2613] [Feature request]
Is it possible to create an option to disable the email notifications that are sent out from the developer portal? We would like to disable individual emails since the only ones we have use for are password and email resets. This would be really key functionality to add. Thanks.

**Comment by Mary-Kwiatkowski** [comment]
Why is there no option on this page to disable a type of email?
![emails](https://github.com/Azure/api-management-developer-portal/assets/30268437/970936fa-6858-41d0-a42a-33fa04313116)

**Comment by alexbenitez** [comment]
Is there any update regarding this limitation?

**Comment by DarinMacRae** [comment]
We would also like to disable all internal APIM notifications and use our own email system. It seems like, at least, we should be able to disable these by disabling the Developer Portal a la (https://learn.microsoft.com/en-us/answers/questions/2126341/how-to-unpublish-api-management-developer-portal).
